tests/Unit/Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/Test_DampedHarmonicGaugeQuantities.cpp

```diff
@@ -148,10 +148,34 @@ tnsr::a<DataVector, SpatialDim, Frame> wrap_DampedHarmonicHCompute(
     const double t, const double t_start, const double sigma_t,
     const tnsr::I<DataVector, SpatialDim, Frame>& coords,
     const double sigma_r) noexcept {
-  return GeneralizedHarmonic::gauges::DampedHarmonicHCompute<
-      SpatialDim, Frame>::function(gauge_h_init, lapse, shift,
-                                   sqrt_det_spatial_metric, spacetime_metric, t,
-                                   t_start, sigma_t, coords, sigma_r);
+  tnsr::a<DataVector, SpatialDim, Frame> gauge_h{};
+  tnsr::ab<DataVector, SpatialDim, Frame> d4_gauge_h{};
+  const auto dgauge_h_init =
+      make_with_value<tnsr::ab<DataVector, SpatialDim, Frame>>(lapse, 0.);
+  const auto spacetime_unit_normal_one_form =
+      gr::spacetime_normal_one_form<SpatialDim, Frame, DataVector>(lapse);
+  const auto pi =
+      make_with_value<tnsr::aa<DataVector, SpatialDim, Frame>>(lapse, 0.);
+  const auto phi =
+      make_with_value<tnsr::iaa<DataVector, SpatialDim, Frame>>(lapse, 0.);
+  auto inverse_spatial_metric =
+      make_with_value<tnsr::II<DataVector, SpatialDim, Frame>>(lapse, 0.);
+  for (size_t i = 0; i < SpatialDim; ++i) {
+    inverse_spatial_metric.get(i, i) = 1.;
+  }
+  GeneralizedHarmonic::gauges::damped_harmonic(
+      make_not_null(&gauge_h), make_not_null(&d4_gauge_h), gauge_h_init,
+      dgauge_h_init, lapse, shift, spacetime_unit_normal_one_form,
+      sqrt_det_spatial_metric, inverse_spatial_metric, spacetime_metric, pi,
+      phi, t, coords, 1., 1.,
```

Comment:
Would the test be more thorough if the values passed in weren't so many repeated values?

Reply:
Yes, I will be doing that in a followup PR. I have a commit for it already. 1 or 2 PRs out since I'm trying to keep the PRs below adding 300 lines. Is that okay? There's a lot of cleanup needed for the DH code and I'm trying to throw it at you slowly :)

Reply:
Yeah, I'm fine with taking this in small doses :P